app/js/helpers/init-points.babel.js

```diff
@@ -3,7 +3,7 @@ import initPoint from './init-point';
 export default (points) => {
 
   const newPoints = [];
-  for (var i = 0; i < points.length; i++) {
+  for (var i = 0, max = points.length; i < max; i++) {
```

Comment:
Its just best practices when it's cheap! I believe it does not have that much of an effect here since there are not so many iterations I suppose!